source/_sass/infeedl.scss

```diff
@@ -93,12 +97,27 @@ h1, h2, h3, h4, h5, h6 {
     margin-left: 8.33333%;
     margin-right: 8.33333%;
   }
+  .sidebar-logo-wrap{
+    width:100%;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Colon after property should be followed by one space